### PR TITLE
chore(wallet): hide network preferences in watched and make multichai…

### PR DESCRIPTION
**Pull request by J-Son89**
A minor pr to clean up the app a bit.

Makes multichain tab the first on the receive screen:

Hides network preferences button on watch only address for now as this is a low priority issue to add this feature.

We can revisit if needed for 2.30 soon 👍

### CODE REVIEW

**Comment by J-Son89** [comment]
I think it can skip manual QA as the changes are very small in scope and pretty trivial. Will do a smoke test myself 👍

**Review by OmarBasem — APPROVED** [review]

File: src/status_im/contexts/wallet/account/share_address/view.cljs [hunk 1]
```diff
@@ -50,7 +50,7 @@
 (defn view
   []
   (let [padding-top         (:top (safe-area/get-insets))
-        wallet-type         (reagent/atom :legacy)
+        wallet-type         (reagent/atom :multichain)
```
Comment (BalogunofAfrica) [inline src/status_im/contexts/wallet/account/share_address/view.cljs:53]: Shouldn't this be a react state according to the new ui guideline? 🤔 https://github.com/status-im/status-mobile/blob/develop/doc/ui-guidelines.md?plain=1#L11-L44

Comment (J-Son89) [reply src/status_im/contexts/wallet/account/share_address/view.cljs:53]: I mean yes and no. This pr is not a refactor pr, as that also leaves rooms for bugs to sneak in. I just want to make the smallest change possible here which is to make this ui right in terms of order of the tabs. New code as a defult should be written with react state ofc 👍

**Review by BalogunofAfrica — APPROVED** [review]

**Comment by J-Son89** [comment]
@status-im/mobile-qa can you check e2e tests when ready.

@mariia-skrypnyk - in this pr I hide the button for network preferences of a watch only account on creation - I do this because it is not implemented yet and we want to hide unfinished functionality for public release 👍 
Once we have that working the issue you described should be in place :)

**Review by vkjr — APPROVED** [review]

**Comment by VolodLytvynenko** [comment]
I will perform e2e one more time

**Comment by VolodLytvynenko** [comment]
hi @J-Son89 !
The e2e results are reviewed, the failed tests are not related to the PR.
PR can proceed to the next stage.

**Pull request merged**